governance-and-automation/manage-custom-metadata.md

```diff
@@ -41,19 +41,21 @@ To start, take the following steps:
 Once the create new metadata dialog opens, fill in the following information.
 * **Metadata Name (1)**; the name of the metadata.
 * **Description (2)**; the description of the metadata for end users.
-* **Enable collaborators to change the created metadata (3)** by checking the Allow collaborators to change this metadata value option
-  * By default, this option is disabled
-  * When enabled, owners will be able to modify metadata on their workspaces
-* Select the **data type (4)** of the new metadata under the Customize Field.
+* **Enable collaborators to change the created metadata (3)** by checking the Allow collaborators to change this metadata value option.
+  * By default, this option is disabled.
+  * When enabled, owners can modify metadata on their workspaces.
+* **Include this metadata in the SharePoint search index (4)**.
+  * **Selecting this checkbox** ensures that editing or provisioning the metadata through Syskit Point **results in an indexed property bag value**.
```

Comment:
ova druga rečenica treba objasniti što znači da je metadata included in SP search index. trenutno mi samo uvodi još jedan pojam (indexed property bag value) koji ne znam što znači (kao end-user)
možda ovaj link može pomoći:
https://blog.kloud.com.au/2018/04/26/how-to-make-property-bag-values-indexed-and-searchable-in-sharepoint-online/